tests/Workleap.DotNet.CodingStandards.Tests/Workleap.DotNet.CodingStandards.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
Could we use nullable reference types?

Also nit: should we use a Directory.Build.props to specify Nullable enable and ImplicitUsings? This could be used for the future EditorConfig generator